From c4287f5dc51090366156beeffbd66a30db39e593 Mon Sep 17 00:00:00 2001
From: Marek Vasut <marex@nabladev.com>
Date: Wed, 7 Jan 2026 17:10:53 +0100
Subject: [PATCH 1/2] ARM: stm32: Use CONFIG_STM32MP15X to discern STM32MP15xx
 on DH STM32MP15xx DHSOM

Use plain CONFIG_STM32MP15X to discern code which is specific to
STM32MP15xx in DH STM32MP1 DHSOM board files.

Upstream-Status: Pending
Signed-off-by: Marek Vasut <marex@nabladev.com>
---
 board/dhelectronics/dh_stm32mp1/board.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/board/dhelectronics/dh_stm32mp1/board.c b/board/dhelectronics/dh_stm32mp1/board.c
index 55526189d5a..90590d14941 100644
--- a/board/dhelectronics/dh_stm32mp1/board.c
+++ b/board/dhelectronics/dh_stm32mp1/board.c
@@ -314,7 +314,7 @@ int board_stm32mp1_ddr_config_name_match(struct udevice *dev,
 			return 0;
 	}
 
-	if (IS_ENABLED(CONFIG_TARGET_DH_STM32MP15X)) {
+	if (IS_ENABLED(CONFIG_STM32MP15X)) {
 		if (ddr3code == 1 &&
 		    !strcmp(name, "st,ddr3l-dhsom-1066-888-bin-g-2x1gb-533mhz"))
 			return 0;
-- 
2.51.0

